downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc

```diff
@@ -101,4 +104,6 @@ sso_keystore_password=''
 # automationedacontroller_ssl_cert=/path/to/automationeda.crt
 # automationedacontroller_ssl_key=/path/to/automationeda.key
 
+automationedacontroller_max_running_activations: 12       # The number of running activations per worker. The default is set to 12.
```

Comment:
Per your recommendation, @jamesmarshall24 , all of the superfluous code info from before has been removed. Only the `automationedacontroller_max_running_activations` has been added now. Is this correct?

Reply:
Removing this line as it is not necessary since it is set to 12 as a default, per @jamesmarshall24. Thanks!

Reply:
Regardless of this line, should not be updated the list of variables in https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_installation_guide/appendix-inventory-files-vars#event-driven-ansible-controller ?

Reply:
@Alex-Izquierdo - I mentioned the update to Appendix A. 5. in the Installation Guide in my discussions with @jamesmarshall24 in our Slack discussions. On March 7, I added a link to appendix in the Conversation tab above so that this update is included in our list of changes. @jamesmarshall24 viewed and approved the update in our Slack discussion as well as the PR (attached to this reply). 

![Screenshot from 2024-03-11 09-11-43](https://github.com/ansible/aap-docs/assets/73364088/ec2ee941-3284-46a1-ba75-fc40c9afbd11)

![Screenshot from 2024-03-11 09-08-07](https://github.com/ansible/aap-docs/assets/73364088/08cce7a0-b337-442b-b473-087d7f8c827d)